p2p/security/noise/session.go

```diff
@@ -55,6 +58,7 @@ func newSecureSession(tpt *Transport, ctx context.Context, insecure net.Conn, re
 		prologue:                  prologue,
 		initiatorEarlyDataHandler: initiatorEDH,
 		responderEarlyDataHandler: responderEDH,
+		connectionState:           network.ConnectionState{NextProto: ""},
```

Comment:
No need to initialize here, this is the default value.

Reply:
OK, thanks for the tip. going with default.